app/views/topics/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2>Topics</h2>
+<p>Add a new topic</p>
+
+<%= form_for @topic, url: {action: "create"}, html: {class: "nifty_form"} do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name, :required => 'required' %>
+  <%= f.submit "Create" %>
```

Comment:
It's always good to be explicit about what is being created (i.e. "Create topic")
I think the default text for a submit button on a POST form is _"Create [model name]"_ , so I think the following will work:
```ruby
<%= f.submit %>
```